src/NATS.Client.Core/NatsSub.cs

```diff
@@ -55,7 +55,7 @@ public abstract class NatsSubBase : INatsSub
             _cts.Token.UnsafeRegister(
                 static (self, _) =>
                 {
-                    ((NatsSubBase) self!).EndSubscription(NatsSubEndReason.Cancelled);
+                    ((NatsSubBase)self!).EndSubscription(NatsSubEndReason.Cancelled);
```

Comment:
Linter complained about this so I went ahead and did it. If revert is preferred LMK.

Reply:
Make sure to run `dotnet format`. That should fix these issues as well.